Sources/IR/Emitter.swift

```diff
@@ -856,16 +867,16 @@ public struct Emitter {
 
   // MARK: l-values
 
-  /// Emits `expr` as a l-value with the specified capability into `module` at the current
-  /// insertion point.
+  /// Emits the IR of `expr` treated as a l-value with given `capability` into `module`, inserting
+  /// instructions at the end of `self.insertionBlock`.
```

Comment:
```suggestion
  /// Inserts the IR of the lvalue `expr`, having the given `capability, into `module` 
  /// at the end of the current insertion block.
```

Maybe not ideal, and I'm not 100% sure I'm conveying the intended meaning (my wording assumes `expr` already has the capability)? We always have to be careful with the word "with" because it has so many interpretations. Is the capability getting inserted into the module along with the IR for `expr`?

It's better to avoid "with" altogether when possible, and consider removing it from the label.